doc/index.rst

```diff
@@ -4,43 +4,120 @@
 
 .. module:: matplotlib
 
-Matplotlib documentation 
+
+Matplotlib documentation
 ------------------------
 
 Release: |release|
 
 Matplotlib is a comprehensive library for creating static, animated,
 and interactive visualizations in Python.
 
-Learn
-=====
-      
-- :doc:`Quick-start Guide <tutorials/introductory/usage>`
-- Basic :doc:`Plot Types <plot_types/index>` and :doc:`Example Gallery <gallery/index>`
-- `Introductory Tutorials <../tutorials/index.html#introductory>`_
-- :doc:`External Learning Resources <resources/index>`
-      
-Reference
-=========
+Installation
+============
+
+.. panels::
+    :card: + install-card
+    :column: col-lg-6 col-md-6 col-sm-12 col-xs-12 p-3
+
+    Working with conda?
+    ^^^^^^^^^^^^^^^^^^^
+
+    Matplotlib is part of the `Anaconda <https://docs.continuum.io/anaconda/>`__
+    distribution and can be installed with Anaconda or Miniconda:
+
+    ++++++++++++++++++++++
+
+    .. code-block:: bash
+
+        conda install matplotlib
+
+    ---
+
+    Prefer pip?
+    ^^^^^^^^^^^
+
+    Matplotlib can be installed via pip from `PyPI <https://pypi.org/project/matplotlib>`__.
+
+    ++++
+
+    .. code-block:: bash
+
+        pip install matplotlib
+
+    ---
+    :column: col-12 p-3
+
+    In-depth instructions?
+    ^^^^^^^^^^^^^^^^^^^^^^
+
+    Installing a specific version? Installing from source? Check the advanced
+    installation page.
+
+    .. container:: custom-button
+
+        :doc:`Installation Guide <users/installing>`
+
+
+Learning Resources
```

Comment:
Reference is a learning resource. Maybe "User documentation" as all this information is directed at our users. Unfortunately, this is close to the existing "User Guide", but the problem is rather that that is a weird catch-all and should probably be renamed (but even if we keep "User Guide" for now, that should not stop us from using "User documentation" here).